README.md

```diff
@@ -1,5 +1,5 @@
-# df2-module-skeleton
+# module-skeleton
```

Comment:
Maybe it is good idea to add some naming prefix?

Reply:
will fix in next iteration